docs/styles.less

```diff
@@ -1571,4 +1572,34 @@ Spark table documentation styles
 */
 .number-picker-component-demo {
     width: 100px;
-}
\ No newline at end of file
+}
+
+/*
+    Blog post code snippets
+*/
+.hljs {
+  background: #f5f5f5;
+  color: #000;
+  font-size: .875rem;
```

Comment:
Add `font-family:  Menlo, Consolas, "Courier New", monospace`

To make the font match our other snippets